aten/src/ATen/cuda/tunable/Tunable.cpp

```diff
@@ -412,6 +431,30 @@ int TuningContext::GetMaxWarmupIterations() const {
   return max_warmup_iterations_;
 }
 
+void TuningContext::SetICacheFlushIterations(int iterations) {
+  icache_flush_iterations_ = iterations;
+}
+
+int TuningContext::GetICacheFlushIterations() const {
+  static const char *env = std::getenv("PYTORCH_TUNABLEOP_ICACHE_FLUSH_ITERATIONS");
+  if (env != nullptr) {
+    return atoi(env);
+  }
+  return icache_flush_iterations_;
+}
+
+void TuningContext::SetRotatingBufferSize(int size) {
+  rotating_buffer_size_ = size;
+}
+
+int TuningContext::GetRotatingBufferSize() const {
+  static const char *env = std::getenv("PYTORCH_TUNABLEOP_ROTATING_BUFFER_SIZE");
+  if (env != nullptr) {
+    return atoi(env) * 1024 * 1024;  // in MiB
```

Comment:
Does it make sense to update rotating_buffer_size_ here ? 
Or, should the env PYTORCH_TUNABLEOP_ROTATING_BUFFER_SIZE set the rotating_buffer_size_  in the constructor, and SetRotatingBufferSize can override it?

Reply:
The env vars have highest priority.  Using the env vars is a different use case than the C++ and python APIs.